docs/docs.json

```diff
@@ -185,6 +179,12 @@
               "v3/how-to-guides/cloud/create-a-webhook",
               "v3/how-to-guides/cloud/troubleshoot-cloud"
             ]
+          },
+          {
+            "group": "Data Applications",
```

Comment:
This group name doesn't feel quite right. Would "Applications" or "APIs" work?

Reply:
i was deciding between this and Applications, so ill go with the latter